modules/CmpdReg/src/client/custom/configuration.json

```diff
@@ -39,7 +39,10 @@
 		"showTotalAmountStored": false,
 		"disableAliasEdit": false,
 		"showLotInventory": false,
-		"disableEditMyLots": false
+		"disableEditMyLots": false,
+		"requireLotNumber": true,
+		"allowManualLotNumber": true,
```

Comment:
So the new default behavior will be to allow a manual lot number, but auto-populate with the next one? Sounds reasonable to me, but it is a default behavior change.

Reply:
Thanks for pointing this out. I checked in the wrong default configs.  I want the default behavior to not change.